spec/commands/newgem_spec.rb

```diff
@@ -116,19 +119,20 @@ def gem_skeleton_assertions(gem_name)
     let(:gem_name) { "test_gem" }
     let(:generated_gem) { Bundler::GemHelper.new(bundled_app(gem_name).to_s) }
 
-    context "git config user.name present" do
+    context "git config github.user present" do
       before do
         execute_bundle_gem(gem_name)
       end
 
       it "contribute URL set to git username" do
         expect(bundled_app("test_gem/README.md").read).not_to include("[USERNAME]")
+        expect(bundled_app("test_gem/README.md").read).to include("github.com/bundleuser")
       end
     end
 
-    context "git config user.name is absent" do
+    context "git config gihhub.user is absent" do
       before do
-        `git config --unset user.name`
+        `git config --unset github.user`
```

Comment:
@danielpclark Here is my take. Code collaboration is difficult, at all times, and it's not interactive, and we all have to wait on each other. Timezones and life and everything. But we can do it!

To get _this_ change merged: focus on the immediate feedback.

If you only change the file where you _have_ to, a second, clean-up-only PR with only "go from backticks to `sys_exec` everywhere in this file" could be a way to get this fix (and the next one) quicker to "done".

Keep changes easy to review.

_Good luck, and thanks for doing this!_